### DISCUSSION TITLE
AutoBuilder v2.2.3

### DISCUSSION THREAD

**Discussion by adaliea** [adaliea/AutoBuilder#40] [Announcements]
## This Release requires you to use > `v2.2.1` of the Robot Side Library!
***v2.2.6** updates the wpilib dependency to 2022.4.2
***v2.2.3** updates the wpilib dependency to 2022.3.2
***v2.2.2** updates the wpilib dependency to 2022.2.1
```groovy
    implementation "com.dacubeking:autobuilder-robot:2.2.26 
````
# **You will need to update some config values if you've previously used an older version (version <= 2.2.0) of the AutoBuilder to ensure that your field image renders correctly!**

**Make sure, under your app config, that you have the following values set**
* Point Scale Factor: 159.83
* Origin X: -598.0
* Origin Y: -1080.0

![image](https://user-images.githubusercontent.com/59785640/211184571-a28d4a67-6916-47a2-9dfc-6c6188be104e.png)

## Bug Fixes:
- Fix reflection of constraints
- Always render the primary robot position on top of all other robot positions

**Full Changelog**: https://github.com/FRC3476/AutoBuilder/compare/v2.2.3...v2.2.4

<hr /><em>This discussion was created from the release <a href='https://github.com/FRC3476/AutoBuilder/releases/tag/v2.2.4'>AutoBuilder v2.2.3</a>.</em>

**Comment by Ronan-B** [comment]
By any chance is it able to work on a Mac OS system?

**Reply by adaliea** [reply]
oh I just saw this. oops... Does it not work on macOS? I'm pretty sure I'm including the native libraries for win/Mac/linux
